### PR TITLE
Remove duplicate call to GetHybridModelZone

### PR DESCRIPTION
**Pull request by mjwitte**
Pull request overview
---------------------
Fix problem with duplicate csv column headers introduced in #7094 .  CI will probably complain about the two newer HybridModel_4Zone test files, because develop will still have the duplicate headers.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add to input rules file for interfaces
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Required documentation updates?
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

File: src/EnergyPlus/ZoneContaminantPredictorCorrector.cc [hunk 1]
```diff
@@ -1316,9 +1316,6 @@ namespace ZoneContaminantPredictorCorrector {
 
         } // ContControlledZoneNum
 
-        // Get the Hybrid Model setting inputs
-        GetHybridModelZone();
-
```
Comment (Myoldmopar) [inline src/EnergyPlus/ZoneContaminantPredictorCorrector.cc:1321]: Whoa.  This wasn't the fix that I expected.  But OK!

Comment (mjwitte) [reply src/EnergyPlus/ZoneContaminantPredictorCorrector.cc:1321]: Not 100% certain this is the proper fix. We could add a flag to make sure this only runs once, but I can't see any reason why this needs to be here when it's already getting called [here](https://github.com/NREL/EnergyPlus/blob/7a131682e99198678e82db06e8d60cb4097b267e/src/EnergyPlus/ZoneTempPredictorCorrector.cc#L1777-L1778) in `GetZoneAirSetPoints`. I suppose it's possible that one could do a hybrid model on CO2 without any HVAC? Then would `GetZoneAirSetPoints` never be called? Hmmm.

Comment (mjwitte) [reply src/EnergyPlus/ZoneContaminantPredictorCorrector.cc:1321]: `GetZoneAirSetPoints` is called from `ManageZoneAirUpdates` which will *always* get called, so I think this is OK.

**Comment by mjwitte** [comment]
Ran the three hybrid zone test files with develop and with this branch. The duplicate columns are gone in this branch, and the results are the same as develop.

File: src/EnergyPlus/ZoneContaminantPredictorCorrector.cc (hunk 1 above)
Comment (Myoldmopar) [reply src/EnergyPlus/ZoneContaminantPredictorCorrector.cc:1321]: I'd suggest moving it even higher to ensure that it gets called once, and only once, every simulation.

Comment (mjwitte) [reply src/EnergyPlus/ZoneContaminantPredictorCorrector.cc:1321]: Any suggestions? A quick scan says the only things that need to be done before this are Zones and ZoneAirMassFlow.EnforceZoneMassBalance but I may have missed something. Well, schedules, too.

Comment (xuanluo113) [reply src/EnergyPlus/ZoneContaminantPredictorCorrector.cc:1321]: I'm not sure but `GetZoneAirSetPoints` already has a `GetZoneAirStatsInputFlag`, would this flag make sure that `GetHybridModelZone` under this be called only once?

**Comment by Myoldmopar** [comment]
I do not know what happened to Windows there, but I can predict the outcome.  2 regression failures due to the column issue.  We've got to get two others in in short order so I'm merging this.

**Pull request merged**